packages/permission-controller/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.0.1]
+### Fixed
+- tuple types from permissions rpc methods ([#1464](https://github.com/MetaMask/core/pull/1464))
```

Comment:
Nit: here is an attempt at cleaning up the wording a bit:
```suggestion
- Fix permissions RPC method types ([#1464](https://github.com/MetaMask/core/pull/1464))
  - The RPC method handlers were mistakenly typed as an array rather than a tuple
```